packages/component/src/Middleware/ScrollToEndButton/ScrollToEndButton.js

```diff
@@ -1,29 +1,55 @@
 import { hooks } from 'botframework-webchat-api';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, { memo, useRef } from 'react';
 
+import useFocusAccessKeyEffect from '../../Utils/AccessKeySink/useFocusAccessKeyEffect';
+import useLocalizeAccessKey from '../../hooks/internal/useLocalizeAccessKey';
 import useStyleSet from '../../hooks/useStyleSet';
+import { useLiveRegion } from '../../providers/LiveRegionTwin';
 
 const { useDirection, useLocalizer, useStyleOptions } = hooks;
 
+const newMessagesAccessKey = 'U u Ü ü';
+
 const ScrollToEndButton = ({ onClick }) => {
+  const focusRef = useRef(null);
   const [{ scrollToEndButton: scrollToEndButtonStyleSet }] = useStyleSet();
   const [{ scrollToEndButtonBehavior }] = useStyleOptions();
   const [direction] = useDirection();
   const localize = useLocalizer();
+  const localizeAccessKey = useLocalizeAccessKey();
 
+  const shouldShowScrollToEndButtonOnUnread = scrollToEndButtonBehavior === 'unread';
   const text = localize(scrollToEndButtonBehavior === 'any' ? 'TRANSCRIPT_MORE_MESSAGES' : 'TRANSCRIPT_NEW_MESSAGES');
 
+  const liveRegionText = localize(
+    'TRANSCRIPT_LIVE_REGION_NEW_MESSAGES_ALT',
+    localizeAccessKey(newMessagesAccessKey),
+    text
+  );
+
+  // Setup and announce new messages button shortcuts
+  useFocusAccessKeyEffect(scrollToEndButtonBehavior === 'any' ? '' : newMessagesAccessKey, focusRef);
```

Comment:
If `scrollToEndButtonBehavior` is `false`, we won't show the button. I think we shouldn't need `useFocusAccessKeyEffect` in that case.